Identrust root download link

tdelmas opened the pull request:
Fix #416
- Emphasis on the fact that Identrust's website is the main place to download
- Give the new commercial name to find the download link on Identrust's website
- Allows to directly download the .p7b and the .pem if really needed

Before:

> Specifically, IdenTrust has cross-signed our intermediate using their
[DST Root CA X3](https://www.identrust.com/certificates/trustid/root-download-x3.html)

After:

> Specifically, IdenTrust has cross-signed our intermediate using their
"DST Root CA X3" (now called "TrustID X3 Root"). [Download "TrustID X3 Root" on identrust.com](https://www.identrust.com/support/downloads) (or, alternatively, you can download a copy here: [.pem](/certs/trustid-x3-root.pem.txt), [.p7b](/certs/trustid-x3-root.p7b)).

bdaehlie commented:
I'm ready to merge this except for the fact that there are 5 static  cert files for the same root being added here, while there are links to only two, and the formats don't match what we normally do (we don't have any other p7b for example). There is also an added file with a "2" at the end, I don't know what that is.

Can you clarify, @tdelmas ?

tdelmas commented:
@bdaehlie sure:

- trustid-x3-root.p7b is the original one, that why I think it's important to have it in the repository and a link to it
- As .cer/.der (binary) and .pem (ASCII) formats are the most commonly used, I've added them, with a link to the .pem.txt (As is was done in https://letsencrypt.org/certificates/)
(I've just renamed the .cer into .der to be coherent with the others)

- trustid-x3-root**2**.p7b is an error, sorry, fixed!

So now we have `.der` `.pem` `.pem.txt` like the other certificates plus the original `.p7b`
